types/src/dkg.rs

```diff
@@ -54,3 +69,63 @@ impl OnChainConfig for DKGState {
     const MODULE_IDENTIFIER: &'static str = "dkg";
     const TYPE_IDENTIFIER: &'static str = "DKGState";
 }
+
+pub trait DKGTrait {
```

Comment:
Why not only defining trx as trait, similar to randmanager? Should also suffice for testing

Reply:
this made it looks like DKGManager takes a "scheme" as a type parameter, not a "transcript type", which is more natural?

Reply:
I don't think there can be difference schemes for such DKG, only the trx may be different (real or mock). But we can keep it like this.

Reply:
I might not have enough context, why do you need Private/Public Params as a type?

Reply:
mainly for easy mocking :)